packages/cli/package.json

```diff
@@ -108,7 +108,7 @@
     "strip-ansi": "6.0.1",
     "tempy": "1.0.1",
     "ts-pattern": "4.0.2",
-    "typescript": "4.5.4"
+    "typescript": "4.6.4"
```

Comment:
Just a thought: have we considered pulling common dev dependencies, like `typescript`, `jest`, `esbuild` and the likes into workspace root so that we don't have to manage their version individually in each package?

Reply:
Renovate does the update management and pnpm is very efficient, so maybe not a big priority here to do. 
It's also more flexible to specify in each package, if one package cannot be upgraded for some reason for example.